src/Components/SmartComponents/CVEs/CVEsTableToolbar.js

```diff
@@ -151,6 +161,7 @@ const CVEsTableToolbarWithContext = ({ context, canEditStatusOrBusinessRisk, can
                             }
                         ),
                         knownExploitFilter(methods.apply, params),
+                        isAppliesToOSEnabled && appliesToOSFilter(methods.apply, params, operatingSystems),
```

Comment:
I think this is where this is happening. When you are on stable (not preview), the "Applies to OS" filter shows up as undefined.

![image](https://github.com/user-attachments/assets/6b2920f1-f6cb-4683-a74d-aef4211a7d31)

Reply:
@johnsonm325 great catch! fixed